### ISSUE TITLE
Incorrect height of list cells

### ISSUE THREAD

**Issue by dlemmermann** [mkpaz/atlantafx#80]
The screenshot below shows how PDFViewFX is trying to render the page thumbnails in a ListView on the left-hand side. For some reason the cell height remains fixed with a low value instead of adjusting to the height of the thumbnails when using AtlantaFX. PDFViewFX does not explicitly set a height for the cells. 

<img width="1112" alt="Bildschirmfoto 2023-11-21 um 10 53 42" src="https://github.com/mkpaz/atlantafx/assets/9534301/8ffd00dd-a11c-48fe-b456-9fba1cb8a4aa">

**Comment by dlemmermann** [comment]
It is possible to work around this by explicitly add -fx-cell-size: -1 to the pdf-view.css file, but I am worried that AtlantaFX changed the default behaviour.

**Comment by mkpaz** [comment]
Yes, that's correct. Table cells have a fixed height by default. To set any custom content other than string, the cell size must be reset. This should be specified explicitly in the doc.
